docs/guide/docs/infrastructure/devtools.md

```diff
@@ -0,0 +1,264 @@
+---
+id: devtools
+title: Developer Tools & Techniques
+---
+
+## 3rd Party OAuth User Authentication
```

Comment:
I don't agree with moving this here ! This is not a devtool at all, it really is a tutorial

Reply:
Alright let me move it to managing  bots and users.